ipaullly/sendIT
# [#161820120] Add login POST route and token generation

#### What does this PR do?
Creates an endpoint that allows a user to sign into their an account and generate a token
#### Description of Task to be completed?
A user can send a POST request to sign into their account and receives a 200 HTTP status code and generated token
#### How should this be manually tested?
clone repo into a local directory
```git clone --single-branch -b ft-POST-login-authentication-161820120 git@github.com:ipaullly/sendIT.git```
cd into the sendIT directory
activate virtual environment
```source venv/bin/activate```
install dependencies
```pip install requirements.txt```
in the home directory execute the test runner to check whether all tests pass
```pytest```
power up POSTMAN and send a JSON body with email and password to the SignIn API. trying to log in with details of an unregistered user will return an error message.
#### What are the relevant pivotal tracker stories?
#29
